### PR TITLE
Sync with normalize 8.0.0

### LINKED ISSUE #26226: on syncing with normalize.css 8.0.0
Congrats on release of 4.1.0!

Not sure if you want to sync to normalize.css or not, and/or bootstrap/reboot may have a different support matrix.  

I'm able to make a PR for some time (and I'm maybe not the best person since I'm not well versed in the nodejs toolchain), but here's my notes. 

The change log for version 8 of normalize.css is

```
8.0.0 (February 2, 2018)

Remove support for older browsers Android 4, lte IE 9, lte Safari 7.
Don't remove search input cancel button in Chrome/Safari.
Form inputs inherit font-family.
Fix text decoration in Safari 8+.
```

It's doesn't map 1 to 1 with reboot anymore but looking that the diff between v7 and v8, here's the relevant changes.  It's mostly just deleting some selectors.

```
git diff 7.0.0 8.0.0 -- normalize.css
```

```diff
 /**
  * 1. Correct the line height in all browsers.
- * 2. Prevent adjustments of font size after orientation changes in
- *    IE on Windows Phone and in iOS.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
  */
 
 html {
   line-height: 1.15; /* 1 */
-  -ms-text-size-adjust: 100%; /* 2 */
   -webkit-text-size-adjust: 100%; /* 2 */
 }
```

```diff
-/**
- * Add the correct font style in Android 4.3-.
- */
-
-dfn {
-  font-style: italic;
-}
-
```

```diff
-/**
- * Hide the overflow in IE.
- */
-
-svg:not(:root) {
-  overflow: hidden;
-}
-
```

```diff
 /**
- * 1. Prevent a WebKit bug where (2) destroys native `audio` and `video`
- *    controls in Android 4.
- * 2. Correct the inability to style clickable types in iOS and Safari.
+ * Correct the inability to style clickable types in iOS and Safari.
  */
 
 button,
-html [type="button"], /* 1 */
+[type="button"],
 [type="reset"],
 [type="submit"] {
-  -webkit-appearance: button; /* 2 */
+  -webkit-appearance: button;
 }
```

```diff
 /**
- * Remove the inner padding and cancel buttons in Chrome and Safari on macOS.
+ * Remove the inner padding in Chrome and Safari on macOS.
  */
 
-[type="search"]::-webkit-search-cancel-button,
 [type="search"]::-webkit-search-decoration {
   -webkit-appearance: none;
 }
```

comments welcome!

n

### PR DESCRIPTION
**Pull request by Varunram**
diff credits: @client9

Fixes #26226

### CODE REVIEW

File: scss/_reboot.scss [hunk 1]
```diff
@@ -274,11 +269,6 @@ img {
   border-style: none; // Remove the border on images inside links in IE 10-.
 }
 
-svg:not(:root) {
```
Comment (XhmikosR) [inline scss/_reboot.scss]: I'm not sure this isn't needed any more. We need to verify it.

**Comment by XhmikosR** [comment]
@Varunram: can you rebase please?

**Comment by Varunram** [comment]
Thanks! Will do shortly.

**Comment by Varunram** [comment]
should be good now :)

**Comment by XhmikosR** [comment]
Ping @mdo @MartijnCuppens @andresgalante

**Comment by MartijnCuppens** [comment]
I think we can also remove this:
https://github.com/twbs/bootstrap/blob/5aed968750177ec35f99dc03e71eec3632beb844/scss/_reboot.scss#L43-L47
But keep the `main`, see https://github.com/necolas/normalize.css/commit/004d58b2f2e0ac3d03d075f8de46ce7c8234742f#diff-bb3dde41d97f19be8ab7b4780a915d5e

**Comment by XhmikosR** [comment]
We still sort of support Android 4.4.

**Comment by XhmikosR** [comment]
So perhaps we should add a TODO comment to remove that in v5.

**Review by mdo — APPROVED** [review]

**Comment by MartijnCuppens** [comment]
Shall we add the TODO then?

**Comment by XhmikosR** [comment]
You can add it if you want @MartijnCuppens.

**Review by MartijnCuppens — APPROVED** [review]

**Pull request merged**